src/Http/Livewire/TeamTransferForm.php

```diff
@@ -83,15 +86,25 @@ public function transferTeam(TransferTeam $transferrer)
             ]);
         }
 
+        $teamMember = Jetstream::findUserByEmailOrFail($this->transferTeamForm['email']);
+
+        if ($this->team->hasUser($teamMember)) {
+            $this->dangerBanner('You cannot transfer team ownership to users outside of this team.');
```

Comment:
Updated string to read a little better.
 
```suggestion
            $this->dangerBanner('You cannot transfer team ownership to a user outside of this team.');
```